src/App.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import './index.css';
+import './App.css';
+import words from './words';
+import images from './images';
+import SoundEffect from './SoundEffect';
+import zvuk1 from './zvuk1.mp3';
+import zvuk2 from './zvuk2.mp3';
+
+const App = () => {
+    const [mainWord, setMainWord] = useState(null);
+    const [randomWords, setRandomWords] = useState([]);
+    const [showMainWord, setShowMainWord] = useState(false);
+    const [showAdditionalWords, setShowAdditionalWords] = useState(false);
+    const [userAnswer, setUserAnswer] = useState('');
+    const [feedback, setFeedback] = useState('');
+    const [mainImage, setMainImage] = useState(null);
+    const winnerSound = new Audio(zvuk1);
+    const lostSound = new Audio(zvuk2)
+
+    const newChangeWord = () => {
+        const index = Math.floor(Math.random() * words.length);
+        setMainWord(words[index]);
+        setMainImage(images[words[index]]);
+        setShowMainWord(true);
+        setShowAdditionalWords(false); 
+        setFeedback('');
+    };
+
+    const fourWords = () => {
+        if (mainWord) {
+            const filteredWords = words.filter(word => word !== mainWord);
+            const additionalWords = getRandomWords(filteredWords, 3); 
+            const allWords = [mainWord, ...additionalWords];
+            const shuffledWords = allWords.sort(() => 0.5 - Math.random());
+            setRandomWords(shuffledWords);
+            setShowAdditionalWords(true);
+        }
+    };
+
+    const getRandomWords = (wordsArray, count = 4) => {
+        const shuffled = [...wordsArray].sort(() => 0.5 - Math.random());
+        return shuffled.slice(0, count);
+    };
+ 
+   const handleAnswer = (word) => {
+        setUserAnswer(word);
+        if (word === mainWord) {
+            setFeedback('Правильно!');
+            winnerSound.play();
+        } else {
+           setFeedback('Неправильно, попробуйте ще раз.');
```

Comment:
Again online text

Reply:
src\inlineText.js
here is the entire text for the application

Reply:
Here also:
`}	else {
	setFeedback(incorrectFeedback);
}
   };`